### DISCUSSION TITLE
How do I install Oauth2?

### DISCUSSION THREAD

**Discussion by lexuanquynh** [meysamhadeli/shop-golang-microservices#87] [Q&A] [answered]
I have tried adding the token ID and secret but I am not sure if they are correct.
If possible, could you please instruct me to add these parameters.
Thank you very much.

**Comment by lexuanquynh** [comment]
Currently, I do not know how to generate access_token and refresh_token.
Any help would be appreciated.

**Reply by meysamhadeli** [reply, marked as answer]
In my project, I use this [go-oauth2/oauth2](https://github.com/go-oauth2/oauth2) library to implement the OAuth2 standard.
You can clone the sample of this repository and try it for more understanding based on documentation. It has an implementation for generate token, refresh token, and other things that you need for security purposes.
